code/ship/ship_flags.h

```diff
@@ -271,7 +271,7 @@ namespace Ship {
     // ignore_count, reinforcement, no_arrival_music, no_arrival_message, no_arrival_warp, no_departure_warp,
 	// same_arrival_warp_when_docked, same_departure_warp_when_docked, no_dynamic, and nav_carry_status
     // Should that change, bump this variable and make sure to make the necessary changes to parse_wing (in missionparse)
-#define PARSEABLE_WING_FLAGS 9
+#define PARSEABLE_WING_FLAGS 10
```

Comment:
Did you add a flag somewhere? Why was this bumped?

Reply:
There are currently 10 parseable flags.  The number was wrong.